apps/payments/next/app/global.css

```diff
@@ -2,403 +2,105 @@
 @tailwind components;
 @tailwind utilities;
 
-html {
-  -webkit-text-size-adjust: 100%;
-  font-family: ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont,
-    Segoe UI, Roboto, Helvetica Neue, Arial, Noto Sans, sans-serif,
-    Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji;
-  line-height: 1.5;
-  tab-size: 4;
-  scroll-behavior: smooth;
-}
 body {
-  font-family: inherit;
-  line-height: inherit;
-  margin: 0;
-}
-h1,
-h2,
-p,
-pre {
-  margin: 0;
-}
-*,
-::before,
-::after {
-  box-sizing: border-box;
-  border-width: 0;
-  border-style: solid;
-  border-color: currentColor;
-}
-h1,
-h2 {
-  font-size: inherit;
-  font-weight: inherit;
-}
-a {
-  color: inherit;
-  text-decoration: inherit;
-}
-pre {
-  font-family: ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas,
-    Liberation Mono, Courier New, monospace;
-}
-svg {
-  display: block;
-  vertical-align: middle;
-  shape-rendering: auto;
-  text-rendering: optimizeLegibility;
-}
-pre {
-  background-color: rgba(55, 65, 81, 1);
-  border-radius: 0.25rem;
-  color: rgba(229, 231, 235, 1);
-  font-family: ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas,
-    Liberation Mono, Courier New, monospace;
-  overflow: scroll;
-  padding: 0.5rem 0.75rem;
+  @apply text-base font-body bg-grey-10 text-grey-900;
+  @apply flex flex-col min-h-[100vh] items-center justify-center;
```

Comment:
L7 isn't needed.

Reply:
I got rid of a couple of `div`'s in the `layout.tsx` which now makes this necessary to keep things justified center.

Reply:
It doesn't make sense to make this a `flex` and then `flex-col`, so let's use `grid` instead and keep `justify-center`. Also, we don't need two `@apply` lines.

`@apply grid justify-center text-base font-body bg-grey-10 text-grey-900;`

Note: see `[offeringId]/checkout/page.tsx` for additional adjustments.

Reply:
Sweet, reworked it a little with your suggestions, and was able to simplify it and get rid of another `<div>`. Nice!